internal/copier/readme/af-component-agent.md

```diff
@@ -1,64 +1,38 @@
-    </a>
-    <a href="/LICENSE">
-        <img src="https://img.shields.io/github/license/datarobot/af-component-agent" alt="License">
-    </a>
-</p>
-
-The agent template provides a set of utilities for constructing a single or multi-agent workflow using frameworks such
-as Nvidia NAT, CrewAI, LangGraph, LlamaIndex, and others. The template is designed to be flexible and extensible, allowing you
-to create a wide range of agent-based applications.
-
-The Agent Framework is component from the [DataRobot App Framework Studio](https://github.com/datarobot/app-framework-studio)
+The DataRobot agent component template provides a set of utilities for constructing a single or multi-agent workflow using frameworks such as Nvidia NAT, CrewAI, LangGraph, LlamaIndex, and others. The template is designed to be flexible and extensible, allowing you to create a wide range of agent-based applications.
 
+This component enables you to build intelligent agent workflows that can handle complex tasks, coordinate multiple agents, and integrate with various AI frameworks. Whether you're building a single-agent application or a sophisticated multi-agent system, this template provides the foundation and utilities needed to get started quickly.
 
 ## Getting Started
 
-To use this template, it expects the base component https://github.com/datarobot/af-component-base has already been
-installed. To do that first, run:
+To use this template, first install the base component [af-component-base](https://github.com/datarobot/af-component-base) if it hasn't already been installed. Run the following command to copy the template:
+
 ```bash
 uvx copier copy https://github.com/datarobot/af-component-base .
 # uvx copier copy git@github.com:datarobot/af-component-base.git .
 ```
 
-To add the agent component to your project, you can use the `uvx copier` command to copy the template from this repository:
+To add the agent component to your project, run the following command to copy the template:
+
 ```bash
-uvx copier copy https://github.com/datarobot/af-component-agent .
-# uvx copier copy git@github.com:datarobot/af-component-agent.git .
+uvx copier copy https://github.com/datarobot-community/af-component-agent .
+# uvx copier copy git@github.com:datarobot-community/af-component-agent.git .
 ```
 
-To update an existing agent template, you can use the `uvx copier update` command. This will update the template files
+To update an existing agent template, run the following command to update the template files:
+
 ```bash
-uvx copier update -a .datarobot/answers/agent-{ agent_app }.yml -A
+uvx copier update -a .datarobot/answers/agent-{{ agent_app }}.yml -A
 ```
 
-
 ## Developer Guide
-Please see the [Development Documentation](/docs/development.md).
 
+Please see the [development documentation](/docs/development.md) for more information on how to develop the agent component template.
```

Comment:
I think this will need to be a fully qualified link:
```suggestion
Please see the [development documentation](https://github.com/datarobot-community/af-component-agent/blob/main/docs/development.md) for more information on how to develop the agent component template.
```